spec/features/trainees/creating_a_new_trainee_spec.rb

```diff
@@ -15,13 +15,12 @@
 private
 
   def when_i_am_viewing_the_list_of_trainees
-    @index_page ||= PageObjects::Trainees::Index.new
-    @index_page.load
+    trainee_index_page.load
```

Comment:
Why are we replacing SitePrism pages?

Reply:
We're not. There's a lot of duplicate code in the feature specs. We can DRY it up a bit using some modules. Example: `spec/support/features/trainee_steps.rb`.